modules/acs-cloud-responsibilities.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * cloud_service/acs-cloud-responsibility-matrix.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-cloud-responsibilities_{context}"]
+= Shared responsibilities for {product-title-managed-short}
+
+While Red Hat manages the {product-title-managed-short} services, also referred to as Central services, the customer has certain responsibilities.
+
+[cols="2,1,1",options="header"]
+|===
+
+|Resource or action
+|Red Hat responsibility
+|Customer responsibility
+
+
```

Comment:
Throughout the following table, unless a word must remain all lowercase for some reason, use the initial cap for each item. For example:
`* Platform monitoring`
`* Software updates`
`* High availability`